Amqp.V1/Akka.Streams.Amqp.V1/NamedQueueSinkSettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Akka.IO;
+using Amqp;
+using Akka.Serialization;
+
+namespace Akka.Streams.Amqp.V1
+{
+
+    public class NamedQueueSinkSettings<T> : IAmpqSinkSettings<T>
+    {
+        private readonly Session session;
+        private readonly string linkName;
+        private readonly string queueName;
+        private readonly Serializer serializer;
```

Comment:
Why not CamelCase for all these prop names? Is it something related to serialization?